superset-frontend/packages/superset-ui-chart-controls/src/sections/sections.tsx

```diff
@@ -38,6 +38,19 @@ export const legacyTimeseriesTime: ControlPanelSectionConfig = {
   ],
 };
 
+export const echartsTimeseriesTime: ControlPanelSectionConfig = {
```

Comment:
Would it make to call this `genericTime` or `genericTimeSection`? Just to decouple it more from the ECharts plugin.

Reply:
alright, I will change it to `genericTime`, but it might be removed after move `granularity_sqla` and `time_range` to the `filter`.